src/parser/open-api/v3/securityParser.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Extracts any security schemes from the given Open API V3 definition file.
+ * Security schemes are defined in `components.securitySchemes`.
+ * The API defines what security applies at either the top level (`security`)
+ * or per operation.
+ *
+ * @param {Object} definition
+ * @return {Object}
+ */
+export function getSecurity (definition) {
```

Comment:
This method is called `getSecurity`, but it returns `securityDefinitions`. This could be confusing.